packages/core/src/Site/index.js

```diff
@@ -1269,7 +1270,9 @@ class Site {
           })),
       };
 
-      fs.outputJsonAsync(siteDataPath, siteData)
+      fse.outputJsonAsync();
```

Comment:
👀 

let's remove all of it:
- there's a few redundant mocks for fs-extra-promise still as well;
- the walk-sync mock which relies on it is also redundant
- package.json entries, then updating `package-lock`

Reply:
😓